pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1331,55 +1331,104 @@ func parseIpspec(ipspec *zconfig.Ipspec,
 	config *types.NetworkInstanceConfig) error {
 
 	config.DomainName = ipspec.GetDomain()
-	// Parse Subnet
-	if s := ipspec.GetSubnet(); s != "" {
-		_, subnet, err := net.ParseCIDR(s)
-		if err != nil {
-			return errors.New(fmt.Sprintf("bad subnet %s: %s",
-				s, err))
-		}
-		config.Subnet = *subnet
-	}
-	// Parse Gateway
-	if g := ipspec.GetGateway(); g != "" {
-		config.Gateway = net.ParseIP(g)
-		if config.Gateway == nil {
-			return errors.New(fmt.Sprintf("bad gateway IP %s",
-				g))
-		}
-	}
 	// Parse NTP Server
 	if n := ipspec.GetNtp(); n != "" {
 		config.NtpServer = net.ParseIP(n)
 		if config.NtpServer == nil {
-			return errors.New(fmt.Sprintf("bad ntp IP %s",
-				n))
+			errStr := fmt.Sprintf("bad ntp IP %s", n)
```

Comment:
You don't need to change these. If you are going to change them they should become fmt.Errorf and nothing else